server/hl7/hl7.route.js

```diff
@@ -13,6 +13,10 @@ router.route('/files')
   /** GET /api/hl7/files - Retrieves all user files */
   .get(hl7Ctrl.getUserFiles);
 
+router.route('/files/parsedMessages/:fileId')
```

Comment:
lets change this route to `/files/:fileId/messages/`

We want the route structure to be a resource-centric tree - so the caller specifies that they care about `files` then they specify which file ( `:fileId`) then they specify that they want the `messages` in that file. This follows a nice hierarchy. Also I generally avoid upper case anything in HTTP URLs, and `parsed` seems unnecessary because I don't think we'll ever want to GET unparsed messages.

Duckduckgo "REST" and what it stands for to get some context on why I am suggesting this structure

Reply:
I might have led @amathew8 astray here. I think I suggested `/pasrsedmessage/` but valid point to take it out.

Reply:
I mean I basically agree, I just suggest `messages` after the `fileId` instead of before to make it more hierarchical 🤷‍♂

Reply:
Yes, that makes sense.